source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
+      hosted in :atlas:`MongodDB Atlas </getting-started?jmp=docs>`, 
+      use that cluster as the source cluster. If you don't have a
+      cluster to work with, you will need to create one.
+
+      You will need a destination cluster that has the same topology
+      as your source cluster. If the source cluster is :ref:`sharded
+      <sharding-background>`, the destination cluster must be sharded
+      in the same way. If the source cluster is a :ref:`replica set
+      <replication>` with 3 nodes, the destination cluster must also be
+      a replica set with 3 nodes.
+
+      The source and destination clusters must be at least MongoDB 6.0.
+      If your clusters are self-managed, they must be MongoDB
+      Enterprise clusters. {+c2c-product-name+} is not supported on
+      MongoDB Community Edition.
+
+      Creating a cluster is beyond the scope of this guide. If you need
+      help, refer to the documentation to create an :ref:`Atlas cluster
+      <create-new-cluster>` or to create a :ref:`self-managed cluster
+      <server-replica-set-deploy>`.
+
+   .. step:: Define administrative users
+
+      If either cluster is hosted in Atlas, or if either of them
+      requires authentication, you must create a database user that has
+      permissions in both clusters. 
+
+      Source Cluster Authentication Requirements
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      The source user must have the following roles:
+
+      - :authrole:`readAnyDatabase` role
+      - :authrole:`clusterMonitor` role
+      - :authrole:`backup` role
+
+      In addition, the source user must be able to:
+
+      - Run the :dbcommand:`getParameter` command
+
+      If the source cluster is hosted in Atlas, the user must have the
+      :ref:`Atlas admin <atlas-user-privileges>` role. The user must
+      also be able to :ref:`read the oplog <ref-atlas-oplog>`.
+
+      Destination Cluster Authentication Requirements
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      If the destination cluster is hosted in Atlas, the user must have
+      the :ref:`Atlas admin <atlas-user-privileges>` role.
+
+      Cluster Authentication Notes
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      - To add an Atlas user, see: :ref:`mongodb-users`.
+      - To add a user to a self-managed cluster, see: :ref:`create-users`.
+      - To verify user permissions, run :method:`db.getUser()`.
+
+   .. step:: Download and install ``mongosync``
+
+      :ref:`c2c-mongosync` is the tool that connects the source and
+      destination clusters. You can host ``mongosync`` on its own
+      hardware, ``mongosync`` does not have to run on the hardware that
+      hosts your MongodDB clusters.
+
+      To install ``mongosync``:
+      
+      #. Locate a host server for the ``mongosync`` executable that has
+         network connectivity to your source and destination clusters.
+      #. Go to the {+mdb-download-center+}.
```

Comment:
can this redirect to https://www.mongodb.com/try/download/mongosync. This will be live 8AM Tuesday Jun 7.

Please also double-check the ones in the installation page